plugins/woocommerce-blocks/assets/js/extensions/shipping-methods/pickup-location/utils.ts

```diff
@@ -54,9 +54,10 @@ export const getInitialSettings = (): ShippingMethodSettings => {
 	const settings = hydratedScreenSettings.pickupLocationSettings;
 
 	return {
-		enabled: settings?.enabled
-			? settings?.enabled === 'yes'
-			: defaultSettings.enabled,
+		enabled:
+			typeof settings?.enabled !== 'undefined'
```

Comment:
Given that this is a boolean, couldn't we check against that type?

```suggestion
			typeof settings?.enabled == 'boolean'
```

Reply:
I assume the intention here is:
- if it's set - get the value even if it's `false`
- if not (undefined) - take the default.

So rather than changing the type check here, `enabled` should be marked optional in line 43.

EDIT: 🤦 I misread Niels' comment. Nevermind.

Reply:
Great suggestion, thanks Niels